exercises/variable-length-quantity/.meta/gen.go

```diff
@@ -0,0 +1,115 @@
+// +build ignore
```

Comment:
we don't need to `build ignore` generators anymore, now that they're in the separate directory, right? They originally were to stop them from being caught when we `go test`, but now no longer necessary.

Doesn't need to be in this PR, maybe I'll make a separate one to remove them.